reference_data/management/tests/update_gencode_2_3_tests.py

```diff
@@ -16,7 +19,7 @@
 
 GTF_DATA = [
     # Comment
-    '#description: evidence-based annotation of the human genome, version 31 (Ensembl 97), mapped to GRCh37 with gencode-backmap\n',
+    '#description: evidence-based annotation of the human genome, version 31 (Ensembl 97), mapped to GRCh37 with gencode-backmap, uni\xe7\xf8de\n',
```

Comment:
It is likely there are unicodes in the descriptions. It doesn't have now but it can have in the future.
Since the `update_gencode` shares the code with `update_dbnsfp_gene`, it is easy to make them all support unicode data.

Reply:
these has been passing in python 2 for years without unicode support

Reply:
Keep unicode in test data to make sure it works when there are unicode chars in the downloaded data.

Reply:
Can you only keep it in the files that ever have unicode? So dbnsfp but not the rest

Reply:
Keep the unicode chars in dbnsfp and hpo data.